External/FEXCore/Source/Interface/Core/JIT/Arm64/MemoryOps.cpp

```diff
@@ -1429,6 +1429,68 @@ DEF_OP(VStoreVectorMasked) {
   }
 }
 
+DEF_OP(VBroadcastFromMem) {
```

Comment:
This IR operation needs to have a half-barrier after the load operation when TSO is enabled.
Take a look at that LoadMemTSO operation under the vector class handling to see how it works.
```cpp
    // Half-barrier.
    dmb(FEXCore::ARMEmitter::BarrierScope::ISHLD);
```

InstCountCI doesn't test with TSO enabled yet so this wouldn't be visible there.

Reply:
Added it. Is the way I did it a correct way of implementing that functionality, or is a completely separate IR op necessary?

Reply:
This works fine, no need for another IR operation in this case.